script.js

```diff
@@ -0,0 +1,2 @@
+const menu = document.getElementById('desktop-nav');
+document.querySelector('.menu-bars').addEventListener('click', () => { menu.style.display = 'flex'; });
```

Comment:
- [ ] This code is not enough to implement the navbar functionality for the mobile navbar. If will recommend you try again yourself, but if you are still having trouble implementing the same, i would suggest having a look at [this](https://www.w3schools.com/howto/howto_js_mobile_navbar.asp)